sql/firefox_accounts_derived/fxa_auth_events_v1/metadata.yaml

```
@@ -0,0 +1,10 @@
+friendly_name: FxA Auth Events
+description: Selected Amplitude events extracted from FxA auth server logs
+owners:
+  - jklukas@mozilla.com
+labels:
+  application: fxa
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_fxa_events
```

https://github.com/mozilla/telemetry-airflow/blob/caa17a72b905adc8e975d9310814032855abea87/dags/fxa_events.py#L30
```suggestion
  dag_name: bqetl_fxa_events
  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']
```